devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -93,7 +93,7 @@ def ping() {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig() + zigbee.levelConfig()
+    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig(0, 300) + zigbee.levelConfig()
```

Comment:
Is it true that the checkInterval has not changed in relation to a prior reporting configuration like it has with some of the other devices? Just wondering why you're not handling the configure reporting response in this DTH like in the others.